Magento2/Sniffs/Exceptions/TryProcessSystemResourcesSniff.php

```diff
@@ -0,0 +1,93 @@
+<?php
+/**
+ * Copyright © Magento. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+namespace Magento2\Sniffs\Exceptions;
+
+use function array_slice;
+use PHP_CodeSniffer\Sniffs\Sniff;
+use PHP_CodeSniffer\Files\File;
+
+/**
+ * Detects no try block detected when processing system resources
+ */
+class TryProcessSystemResourcesSniff implements Sniff
+{
+    /**
+     * String representation of warning.
+     *
+     * @var string
+     */
+    protected $warningMessage = 'All resources SHOULD be properly released.';
```

Comment:
Please change message to: `The code MUST be wrapped with a 'try' block if the method uses system resources `.